CommandOptions.php

```diff
@@ -149,4 +149,26 @@ public function strip()
         return $this;
     }
 
+    /**
+     * @param string $type
+     * @return $this
+     */
+    public function interlace($type)
+    {
+        $this->command .= ' -interlace ' . $type;
+
+        return $this;
+    }
+
+    /**
+     * @param Float $radius
+     * @return $this
+     */
+    public function gaussianBlur($radius)
+    {
+        $this->command .= ' -gaussian-blur ' . $radius;
```

Comment:
Please escape the value and validate according to ImageMagick's documentation:
http://www.imagemagick.org/script/command-line-options.php#gaussian-blur